spec/immutable-data-structures.html

```diff
@@ -127,15 +127,22 @@ <h1>Tuple.from ( _items_ [ , _mapFn_ [, _thisArg_ ] ] )</h1>
           1. Let _k_ be 0.
           1. Let _usingIterator_ be ? GetMethod(_items_, @@iterator).
           1. If _usingIterator_ is not *undefined*, then
-            1. Let _adder_ be a new Abstract Closure with parameters (_key_, _value_) that captures (_list_, _mapFn_, _thisArg_, _mapping_, _k_) and performs the following steps when called:
+            1. Let _iteratorRecord_ be ? GetIterator(_items_, ~sync~, _usingIterator_).
+            1. Repeat,
+              1. If _k_ &ge; 2<sup>53</sup> - 1, then
```

Comment:
```suggestion
              1. If _k_ &gt; 2<sup>53</sup> - 1, then
```

If I've ready this right (which I may not have) then 2**52-1 is still a valid [integer-index](https://tc39.es/ecma262/#integer-index) ?

Reply:
I copied this from `Array.from` without really checking if we have a limit for Tuples. I'll verify it on Monday!

Reply:
https://github.com/tc39/ecma262/issues/2578